girder_worker/context/girder_context.py

```diff
@@ -9,7 +9,12 @@ def create_task_job(job_defaults, sender=None, body=None,
 
     from girder.utility.model_importer import ModelImporter
     from girder.api.rest import getCurrentUser
-    from girder_worker.girder_plugin import utils
+    try:
+        # girder v2 worker plugin
+        from girder.plugins.worker import utils
```

Comment:
The purpose of this change is not to fix the tests--it's to maintain compatibility with Girder 2.  If it were difficult to accomplish, I would say we shouldn't bother.  But this will save us a lot of hassle in deployments and testing during the Girder 3 migration period.

Reply:
Ok, seems reasonable. I'll merge this branch if you approve it.